docs/how-tos/integrating-with-contentful.md

```diff
@@ -0,0 +1,119 @@
+---
+sidebar_position: 6
+---
+
+# Integrating with Contentful
+
+It is common for marketing and content teams to want an A/B testing solution with a point-and-click visual editor to make straightforward copy or image optimizations. However, these solutions most often rely on blocking scripts that cause performance degradations—slowing your site down by 200-500 milliseconds in best-case scenarios. You could avoid these performance issues by fully implementing feature flags for marketing experiments—but this will mean marketing has to rely on engineering for every test they want to run.
+
+Using a headless CMS like Contentful, and a lightweight, robust feature flagging SDK, like Eppo, allows the best of both worlds: an easy way for marketing teams to run tests without sacrificing performance. In this tutorial, we will walk through how to set up Eppo’s Node SDK so Marketing teams can run their own experiments by updating [Content](https://www.contentful.com/help/content-model-and-content-type/) defined in Contentful and feature flags in Eppo. 
+
+
+## Setting up Contentful
+
+For this example, imagine we are running an experiment on the Eppo homepage. There are a few places where we might want to experiment on this page:
```

Comment:
I think we're missing an image here

Reply:
You know what, I think i just opted to not include it to make it less 'busy', but thinking about it again it will be helpful to include. I'll put it back in